ex11_ch3.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: get input on amount of money in euros
+amount_of_euros = int(input("How much money in euros do you have? "))
+
+#step2: get input about the exchange rate
+euro_exchange_rate = float(input("What is the current euro exchange rate? "))
+
+#step3: write calculation to convert the euros to US dollars
+dollar_exchange_rate = 1.17
```

Comment:
Can this be a constant??? ;)

Reply:
I thought about that initially. However, I use constants when the value is never going to change, correct? The values will inevitably change for these calculations: the user's input for the number of euros, the current euro conversion rate, and the dollar conversion rate. All of these will change, so I didn't make the currency conversion formula a constant. (But I tried it and got an error.)